### PR TITLE
MOTECH-2502: Audit Field Migration

### PR DESCRIPTION
**Pull request by mgrabowski7**

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: http-agent/src/main/resources/db/migration/V1_MOTECH-2502.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+-- change type of field "responseBody" to text
```
Comment (sebbrudzinski) [inline http-agent/src/main/resources/db/migration/V1_MOTECH-2502.sql]: Shouldn't we have two migration files? One for mysql and one for postgresql?
Is this a correct path for the migration? Aren't they placed in db/migration/mysql and db/migration/default?

File: http-agent/src/main/resources/db/migration/V1_MOTECH-2502.sql [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+ALTER TABLE "HTTP_AGENT_MODULE_HTTPACTIONAUDIT" ALTER COLUMN "responseBody" TYPE text;
```
Comment (sebbrudzinski) [inline http-agent/src/main/resources/db/migration/V1_MOTECH-2502.sql]: From what I recall this was supposed to be mediumtext or longtext in mysql. Can you check what type is created on clean install w/o the migration when we set the field type as text and use that db type in migration?

File: http-agent/src/main/java/org/motechproject/http/agent/domain/HTTPActionAudit.java [hunk 3]
```diff
@@ -17,7 +17,7 @@
     @Field
     private String request;
 
-    @Field
+    @Field(type = "TEXT")
```
Comment (sebbrudzinski) [inline http-agent/src/main/java/org/motechproject/http/agent/domain/HTTPActionAudit.java]: We have a constant for that already.

File: http-agent/src/main/resources/db/migration/V1_MOTECH-2502.sql (hunk 1 above)
Comment (sebbrudzinski) [inline http-agent/src/main/resources/db/migration/V1_MOTECH-2502.sql]: @mgrabowski7 Actually, this entity has been just recently added, so we can skip writing a specific migration to fix that column type.

**Comment by sebbrudzinski** [comment]
test this please

File: http-agent/src/main/java/org/motechproject/http/agent/domain/HTTPActionAudit.java [hunk 4]
```diff
@@ -17,7 +17,7 @@
     @Field
     private String request;
```
Comment (sebbrudzinski) [inline http-agent/src/main/java/org/motechproject/http/agent/domain/HTTPActionAudit.java:18]: Btw. can we also make the request a TEXT type? 255 characters is not that much and I can easily imagine someone complaining about it soon.

**Comment by sebbrudzinski** [comment]
test this please

**Pull request merged**